tests/storybook-config.test.js

```diff
@@ -12,6 +12,31 @@ const appDir = path.dirname(
 );
 const storybookDistDir = path.resolve(appDir, 'dist-storybook');
 
+const getStorybookElement = async (url, elementSelector) => {
```

Comment:
```suggestion
const getElementWithinStorybookFrame = async (storybookUrl, elementSelector) => {
```

to make it more clear what this function does